src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents the block number in a Person's address in the address book.
```

Comment:
well, it need not be in the address book, right? the current application is an address book. 
try making the comment(s) generic